### PR TITLE
Facade removal

### PR DESCRIPTION
**Pull request by jbrooksuk**

### CODE REVIEW

File: app/Http/Middleware/SetupAlreadyCompleted.php [hunk 1]
```diff
@@ -12,13 +12,41 @@
 namespace CachetHQ\Cachet\Http\Middleware;
 
 use Closure;
+use Illuminate\Config\Repository;
```
Comment (GrahamCampbell) [inline app/Http/Middleware/SetupAlreadyCompleted.php]: should be using the contract

**Comment by GrahamCampbell** [comment]
This isn't really necessary imo.

**Comment by GrahamCampbell** [comment]
When I say I want to avoid using the config facade, what i really mean is accessing the config repository in any way, inline like this.

**Comment by jbrooksuk** [comment]
So then we can close #1440?

**Comment by GrahamCampbell** [comment]
No. There's still some stuff that I don't like. Just injecting the dependencies doesn't fix that, it just adds more code.

**Pull request closed without merging**